src/analyzer/analyzerebur128mit.cpp

```diff
@@ -0,0 +1,93 @@
+#include <analyzer/analyzerebur128mit.h>
+#include <QtDebug>
+
+#include "trackinfoobject.h"
+#include "util/math.h"
+#include "util/sample.h"
+#include "util/timer.h"
+
+static const float kReplayGain2ReferenceLUFS = -18;
+
+AnalyzerEbur128Mit::AnalyzerEbur128Mit(UserSettingsPointer config)
+        : m_pConfig(config),
+          m_initalized(false),
+          m_iBufferSize(0),
+          m_pState(nullptr) {
+}
+
+AnalyzerEbur128Mit::~AnalyzerEbur128Mit() {
+}
+
+bool AnalyzerEbur128Mit::initialize(TrackPointer tio,
+        int sampleRate, int totalSamples) {
+    if (loadStored(tio) || totalSamples == 0) {
+        return false;
+    }
+
+    m_pState = ebur128_init(2u,
+            static_cast<unsigned int>(sampleRate),
+            EBUR128_MODE_I);
+
+    m_initalized = true;
+    return true;
+}
+
+bool AnalyzerEbur128Mit::loadStored(TrackPointer tio) const {
+    int version = m_pConfig->getValueString(
+            ConfigKey("[ReplayGain]", "ReplayGainVersion")).toInt();
+    // WARNING: Do not fix the "analyser" spelling here since user config files
+    // contain these strings.
+    bool analyzerEnabled = ((bool)m_pConfig->getValueString(
+            ConfigKey("[ReplayGain]", "ReplayGainAnalyserEnabled")).toInt()) &&
+            (version == 2);
+    bool reanalyse = m_pConfig->getValueString(
+            ConfigKey("[ReplayGain]", "ReplayGainReanalyze")).toInt();
+
+    if (analyzerEnabled) {
```

Comment:
I actually though the same when implementing it. But I did not do it because of the other pending naming issue. 
It looks like the Analyzer API requires really some clean-ups.